test/networkd-test.py

```diff
@@ -49,6 +49,14 @@
 @unittest.skipIf(networkd_active,
                  'networkd is already active')
 class ClientTestBase:
+    @classmethod
+    def setUpClass(klass):
+        subprocess.check_call(['systemd-analyze', 'set-log-level', 'debug'])
+
+    @classmethod
+    def tearDownClass(klass):
+        subprocess.check_call(['systemd-analyze', 'set-log-level', 'info'])
```

Comment:
This assumes that the previous log level is "info". Is it correct? (or does it matter? :-)